.gitignore

```diff
@@ -0,0 +1,3 @@
+composer.lock
+phpunit.xml
+vendor/
```

Comment:
missing EOL (there are more in the PR)

Reply:
How about adding a `.editorconfig` to the project? This would reduce these kinds of errors.